src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1248,6 +1244,7 @@ function ($a) {
                                 ->children()
                                     ->scalarNode('dsn')->end()
                                     ->scalarNode('serializer')->defaultNull()->info('Service id of a custom serializer to use.')->end()
+                                    ->scalarNode('bus')->defaultNull()->info('The bus name to dispatch received messages to. This must only be set if you have more than one bus.')->end()
```

Comment:
```suggestion
                                    ->scalarNode('bus')->defaultNull()->info('The bus name to dispatch received messages to. This only needs to be set if you have more than one bus.')->end()
```

If I understand the code correctly, it's safe to explicitly specify the bus even if there's only 1 bus, so this sentence is misleading.

Reply:
true, thanks